src/day/day.module.ts

```diff
@@ -15,18 +15,18 @@
  */
 
 import { Module } from '@nestjs/common';
-import { TsWeekController } from './tsweek.controller';
-import { TsWeekService } from './tsweek.service';
+import { DayController } from './day.controller';
```

Comment:
github is saying this is renamed from 
`src/tsweek/tsweek.module.ts` -> `src/day/day.module.ts`

I think it must be getting confused